packages/models-library/src/models_library/services.py

```diff
@@ -661,3 +661,9 @@ class Config:
                 },
             }
         }
+
+
+class UserWithoutServiceAccess(BaseModel):
+    user_id: int
+    service_key: str
+    service_version: str
```

Comment:
```suggestion
class UserWithoutServiceAccess(BaseModel):
    user_id: UserID
    service_key: ServiceKey
    service_version: ServiceVersion
```

Reply:
```suggestion
class UserWithoutServiceAccess(BaseModel):
    user_id: UserID
    service_key: ServiceKey
    service_version: ServiceVersion
```

Also I think I would change the name of that class. I am not sure I get the meaning of it from the name.

and I do not think it should be in that module. Maybe create something like ```api_schemas_catalog.py``` (what do you think @pcrespov?)
what about *UserInaccessibleService* maybe?

Reply:
done :+1: